README.md

```diff
@@ -49,7 +49,7 @@ This will produce a single file (Pax.exe), called an *assembly* in .NET jargon.
 # Writing for Pax
 Packet processers using Pax can be written in any [.NET language](https://en.wikipedia.org/wiki/List_of_CLI_languages).
 They use Pax's API and define one or more functions that handle incoming packets.
-The [examples](https://github.com/niksu/pax/tree/master/examples) included with Pax could help get you going.
+The [examples](examples) included with Pax could help get you going.
```

Comment:
What do you think of this @niksu? I checked it works, and it's more portable, but I feel like the links are slower?